dataikuapi/dss/admin.py

```diff
@@ -1169,6 +1171,38 @@ def delete(self):
         return self.client._perform_empty(
             "DELETE", "/personal-api-keys/%s" % self.id_)
 
+class DSSPersonalApiKeyListItem(DSSTaggableObjectListItem):
+    """An item in a list of datasets. Do not instantiate this class, :meth:`dataikuapi.DSSClient.list_personal_api_keys` or :meth:`dataikuapi.DSSClient.list_all_personal_api_keys`"""
```

Comment:
We are missing a few words.
BTW we have in the 3 classes close to this one 3 different wording to say the same thing. We could at least chose one here and use it for the 3 of them:
> Do not create this object directly, use :meth:`my_method` instead.

WDYT?

Reply:
Good idea. I just aligned all the classes to it.